src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(int radius, String color) {
+        this.radius = radius;
+        this.setColor(color);
+    }
+
+    public void draw() {
```

Comment:
Let's stick to the task description
behavior - we can obtain the area of any figure and are able to draw it. To 'draw' means to print out all information about a figure using System.out.println() (you shouldn't override the toString() method for this).

Also, you need to create interface with calculateArea() method and Figure implements this interface.

Reply:
I didn`t understand this comment: I didn't override the toString() method for that, and also there is no ionformation about calculateArea() in README file, is there? So I don`t understand what do you want me to do here.

Reply:
I was blind, sry, no need to answer this question